src/helm/benchmark/contamination/light_scenario.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass
+from typing import List, Dict, Any
+
+
+@dataclass(frozen=True, eq=False)
+class LightInstance:
+    """
+    A lighter `Instance` with only text fields.
+    """
+
+    input: str
+    """The input"""
+
+    references: List[str]
+    """References that help us evaluate"""
+
+
+@dataclass(frozen=True, eq=False)
+class LightScenario:
+    """
+    A lighter `Scenario`.
+    """
+
+    scenario_spec: Dict[str, Any]
```

Comment:
This is not quite a scenario spec (see above comments) - make it one or just use `run_spec.name`